doc/source/user_guide/label_guide.rst

```diff
@@ -87,7 +87,7 @@ Sorting variable names
 
 .. tabbed:: xarray
 
-  In xarray, subsetting the Datset with a sorted list of variable names will order the Dataset.
+  In xarray, subsetting the Dataset with a sorted list of variable names will order the Dataset.
```

Comment:
```suggestion
  In xarray, subsetting the Datset with a sorted list of variable names will order the Dataset.
```

This is already fixed in #1699 which also changes other things, with this I aim to avoid git condlicts once merged